spec/features/admin/monthly_report_comment_spec.rb

```diff
@@ -27,9 +29,11 @@
       click_on '月報コメントを更新'
     end
 
-    it { expect(page).to have_content(new_comment) }
-    it { expect(current_path).to eq admin_monthly_report_comment_path(comment) }
-    it { expect(comment.reload.comment).to eq new_comment }
+    it 'should update the monthly report comment' do
+    expect(page).to have_content(new_comment)
```

Comment:
__[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->